edit-post/components/layout/style.scss

```diff
@@ -17,12 +17,12 @@
 			top: inherit;
 		}
 
-		.notice {
+		.components-notice {
 			margin: 0 0 5px;
 			padding: 6px 12px;
 			min-height: $panel-header-height;
 
-			.notice-dismiss {
+			.components-notice__dismiss {
 				margin: 5px;
```

Comment:
Can we tweak this margin to make it appear centered by default if the message is in a single line ?

```css
margin: 10px 5px;
```